graph/pull.go

```diff
@@ -428,10 +428,10 @@ func (s *TagStore) pullV2Tag(eng *engine.Engine, r *registry.Session, out io.Wri
 	}
 
 	if verified {
-		out.Write(sf.FormatStatus(repoInfo.CanonicalName+":"+tag, "The image you are pulling has been verified"))
-	} else {
-		out.Write(sf.FormatStatus(tag, "Pulling from %s", repoInfo.CanonicalName))
+		log.Printf("Image manifest verified for %s:%s", repoInfo.CanonicalName, tag)
 	}
+	out.Write(sf.FormatStatus(tag, "Pulling from %s", repoInfo.CanonicalName))
```

Comment:
@icecrime I read it as "It's pulling the image", not the "manifest". So it verifies the manifest first and then pulls the image data. Should we clarify it somehow?